api/integrations/google/gcal-bot

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import sys
+import time
+import datetime
+import optparse
+from six.moves import urllib
+import itertools
+import traceback
+import os
+import httplib2
```

Comment:
Conventionally there is an expected grouping of Python imports: standard library imports, related third party imports and local application/library specific imports:

https://www.python.org/dev/peps/pep-0008/#imports

Also having them in alphabetical order will look good too.